bindinfo/handle.go

```diff
@@ -187,20 +188,25 @@ func (h *BindHandle) AddBindRecord(record *BindRecord) (err error) {
 	if err1 != nil {
 		return err1
 	}
-	record.CreateTime = types.Time{
-		Time: types.FromGoTime(oracle.GetTimeFromTS(txn.StartTS())),
-		Type: mysql.TypeDatetime,
-		Fsp:  3,
-	}
-	record.UpdateTime = record.CreateTime
-	record.Status = Using
+	for i := range record.Bindings {
+		record.Bindings[i].CreateTime = types.Time{
+			Time: types.FromGoTime(oracle.GetTimeFromTS(txn.StartTS())),
+			Type: mysql.TypeDatetime,
+			Fsp:  3,
+		}
+		record.Bindings[i].UpdateTime = record.Bindings[0].CreateTime
```

Comment:
Just out of curiosity. Why we need the `CreateTime`. Seems that the `UpdateTime` is enough.

Reply:
Only for users to know when the bindings were created.

Reply:
OK👌